<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HStimulusOrderSelectionForm</class>
 <widget class="QWidget" name="HStimulusOrderSelectionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelMain">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PreTest Stimulus Order</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbDefault">
     <property name="text">
      <string>Default Order</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rbSelect">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxOrders"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkboxRandomize">
       <property name="text">
        <string>Randomize?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxRandomizationType"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="text">
      <string>No order selected</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rbSelect</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbxOrders</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkboxRandomize</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbxRandomizationType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
